apps/asap-server/src/migrations/1646992836-replace-empty-string-with-null-in-research-output-link.ts

```diff
@@ -0,0 +1,22 @@
+/* istanbul ignore file */
+import { RestResearchOutput } from '@asap-hub/squidex';
+import { Migration } from '../handlers/webhooks/webhook-run-migrations';
+import { applyToAllItemsInCollection } from '../utils/migrations';
+
+export default class MapResearchOutputDeprecatedSubtype extends Migration {
+  up = async (): Promise<void> => {
+    await applyToAllItemsInCollection<RestResearchOutput>(
+      'research-outputs',
+      async (researchOutput, squidexClient) => {
+        if (researchOutput?.data?.link?.iv === '') {
+          await squidexClient.patch(researchOutput.id, {
+            link: undefined,
```

Comment:
@peterstarling is it going to work?
If I remember correctly squidex had some problem with undefined fields

Reply:
@niezgoda I went with what the type system allowed me. What other approach would there be?

Reply:
@niezgoda Afaik the migration runs when the sls is deployed. Shouldn't this throw an error if it doesn't work? cc @peterstarling